### PR TITLE
Port library environment variable from unicorn

### LINKED ISSUE #1035: use system capstone libraries if they are available
The issue is similar with https://github.com/unicorn-engine/unicorn/issues/869

I don't see any no reason why the library is installed 3 times in addition to the system:

```
bash$ equery f capstone-python | grep capstone.so
/usr/lib64/python2.7/site-packages/capstone/ccapstone.so
/usr/lib64/python2.7/site-packages/capstone/lib/libcapstone.so
/usr/lib64/python3.4/site-packages/capstone/lib/libcapstone.so
```
```
bash$ equery f capstone | grep .so
/usr/lib64/libcapstone.so
/usr/lib64/libcapstone.so.4
```

Please consider

### PR DESCRIPTION
**Pull request by rhelmot**
closes #1035

### CODE REVIEW

File: bindings/python/README.txt [hunk 1]
```diff
@@ -8,10 +8,18 @@ compile C code. On Linux, this is usually easy, but on Windows, this involves
...
+inhibit the build by setting the environment variable LIBCAPSTONE_PATH. The
+exact value is not checked, just setting it will inhibit the build. During
+execution, this variable may be set to the path of a directory containing a
+specific version of libcapstone you would like to use.
+
 If you don't want to build your own copy of Capstone, you can use a precompiled
 binary distribution from PyPI. Saying `pip install capstone` should
 automatically obtain an appropriate copy for your system. If it does not, please
-open an issue at https://github.com/aquynh/capstone and tag @rhelmot - they
+open an issue at https://github.com/aquynh/capstone and tag @rhelmot - she
```
Comment (aquynh) [inline bindings/python/README.txt]: cool ;-)

**Comment by aquynh** [comment]
merged, thanks!

**Comment by rumbah** [comment]
This commit breaks the Python bindings...
```python
>>> import capstone
Traceback (most recent call last):
 ...
  File "/usr/local/lib/python2.7/site-packages/capstone/__init__.py", line 240, in <module>
    _path_list = [os.getenv('LIBCAPSTONE_PATH', None),
NameError: name 'os' is not defined
````

**Comment by aquynh** [comment]
i merged, and fixed it already.

**Comment by rumbah** [comment]
Are you sure? It looks like it's still broken in master (aeda70f)...

**Comment by aquynh** [comment]
right, i forgot to fix that in master branch.

fixed, can you pull again now?

**Comment by rumbah** [comment]
Yep, thanks!

**Pull request merged**